app/src/containers/ConnectedSidebar/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1026,105 +1086,247 @@ Array [
 ]
 `;
 
-exports[`components/ConnectedSidebar renders the ConnectedSidebar component 1`] = `
-Object {
-  "children": Array [
-    <div
-      className="ant-layout-sider-children"
+exports[`components/ConnectedSidebar correctly expand users menu 1`] = `
+Array [
```

Comment:
@ashfahan how can we break down this snapshot

Reply:
many of these snapshots are coming from expansion tests @HaseemUlHaq @haseemVD can you guide us better on this ?